demisto_sdk/commands/common/logger.py

```diff
@@ -1,142 +1,205 @@
 import logging
-import os
-import sys
+import logging.config
+import os.path
+from logging.handlers import RotatingFileHandler
 from pathlib import Path
-from typing import Optional
 
-from rich.logging import RichHandler
+from demisto_sdk.commands.common.content_constant_paths import CONTENT_PATH
+
+# from rich.logging import RichHandler
```

Comment:
We don't need it?
If so, let's delete it

Reply:
Done: https://github.com/demisto/demisto-sdk/pull/2707/commits/fccd0f5435300e3be256f450d7faf51c420ec809